common/test/db_cache/bok_choy_schema.sql

```diff
@@ -753,17 +805,6 @@ CREATE TABLE `certificates_generatedcertificate` (
   CONSTRAINT `user_id_refs_id_8e23bfe2` FOREIGN KEY (`user_id`) REFERENCES `auth_user` (`id`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 /*!40101 SET character_set_client = @saved_cs_client */;
-DROP TABLE IF EXISTS `circuit_servercircuit`;
```

Comment:
@Kelketek there are lots of changes in this file, and some of those does not seem related to badges/certificates. Might make sense to take another look if those changes are needed.